src/userpages/home/Home.js

```diff
@@ -70,8 +70,10 @@ class HomeComponent extends React.Component {
 
         let recoveryPathView = []
         recoveryPath.forEach((s, idx) => {
-            if(idx === recoveryPath.length - 1) {
-                recoveryPathView.push(<span key="and">{' and '}</span>)
+            if(recoveryPath.length!==1){
+                if(idx === recoveryPath.length - 1) {
```

Comment:
Although this code will work, a better way would be to merge these two `if` statements. Apply `&&` operator. This will make the code easier to read and compact.